builds/acr-cleanup/acr-registry-cleanup.yaml

```
@@ -34,7 +34,10 @@ parameters:
         repoTagFilters: [".*:^staging-.*"]
       - olderThan: "14d"
         keep: 1
-        repoTagFilters: [".*:^pr-.*", "^labs/.*:.*", "^toffee.*:.*", "^plum.*:.*"]
+        repoTagFilters: [".*:^pr-.*", "^toffee.*:^latest", "^plum.*:^latest"]
```

this should be fine unless there's something being missed:
```suggestion
        repoTagFilters: [".*:^pr-.*"]
```